src/dask_histogram/boost.py

```diff
@@ -46,6 +102,9 @@ class Histogram(bh.Histogram, DaskMethodsMixin, family=dask_histogram):
         type is :py:class:`boost_histogram.storage.Double`.
     metadata : Any
         Data that is passed along if a new histogram is created.
+    split_every : int | bool | None, default None
+        Width of aggregation layers for staged fills.
+        If False, all staged fills are added in one layer (memory intensive!).
```

Comment:
Not sure about this as the default!

Reply:
default is `None` which means it picks up the split-every from the config file! That should be fine, no?

Reply:
OK - I conflated False and None. Perhaps good to mention where the default comes from if left as None